controllers/image.go

```diff
@@ -48,6 +48,10 @@ func (r *OperatorPipelineReconciler) reconcileCertifiedImageStream(ctx context.C
 	stream := newImageStream(key)
 	if IsObjectFound(ctx, r.Client, key, stream) {
 		log.Info("existing certified image stream found")
+		if err := r.updateStatusCondition(ctx, pipeline, certifiedImageStreamAvailable, metav1.ConditionTrue, reconcileSucceeded,
```

Comment:
non-blocking nit: Please don't split these lines like this. The readability goes down, not up.